### PR TITLE
FIX: mayavi figure support

### LINKED ISSUE #1778: BUG in Report.add_figs_to_section
cc @mainakjas 

https://github.com/mne-tools/mne-python/blob/master/mne/report.py#L724

img will not be defined when this block of code get's executed

### PR DESCRIPTION
**Pull request by dengemann**
closes #1778 

cc @agramfort @mainakjas

### CODE REVIEW

File: mne/report.py [hunk 1]
```diff
@@ -701,6 +701,8 @@ def _add_figs_to_section(self, figs, captions, section='custom',
                              image_format='png', scale=None):
         """Auxiliary method for `add_section` and `add_figs_to_section`.
         """
+        from scipy.misc import imread
```
Comment (Eric89GXL) [inline mne/report.py]: This one doesn't need to be nested

**Comment by Eric89GXL** [comment]
Other than my minor comment, LGTM

File: mne/report.py (hunk 1 above)
Comment (mainakjas) [inline mne/report.py]: well, it does because it depends on PIL indirectly

**Comment by mainakjas** [comment]
Also, test fails right now.

**Comment by dengemann** [comment]
@mainakjas this should be fixed with the last commit.

**Comment by agramfort** [comment]
+1 for merge if travis is happy

**Comment by dengemann** [comment]
Thanks for the quick reviews!!

**Pull request merged**